scripts/docker

```diff
@@ -143,6 +143,7 @@ if [ "$CMD" == "test" -o "$CMD" == "hqtest" -o "$CMD" == "py3test" ]; then
     fi
     export COMPOSE_FILE=docker/hq-compose.yml
     export COMPOSE_PROJECT_NAME=hqtest
+    export CITUS_DB="${CITUS_DB:-test_commcare_ucr_citus}"
```

Comment:
Im not totally sure I follow what is going on in this file. Does redefining this here mean that the initial setup will create citus dbs with the name `test_commcare_ucr_citus` in Travis rather than having initial ones with `commcare_ucr_citus`

Reply:
Correct. Not only Travis but any time a `./scripts/docker test|hqtest ...` command is run. This can also be overridden by setting the `CITUS_DB` environment variable.

Reply:
Nice